code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -29,24 +29,31 @@
 	var/trit = . ? .[MOLES] : 0
 	. = air_gases[/datum/gas/hydrogen]
 	var/h2 = . ? .[MOLES] : 0
+	. = air_gases[/datum/gas/freon]
+	var/freon = . ? .[MOLES] : 0
 	if(active_hotspot)
 		if(soh)
 			if(plas > 0.5 || trit > 0.5 || h2 > 0.5)
 				if(active_hotspot.temperature < exposed_temperature)
 					active_hotspot.temperature = exposed_temperature
 				if(active_hotspot.volume < exposed_volume)
 					active_hotspot.volume = exposed_volume
+			if(freon > 0.5)
+				if(active_hotspot.temperature > exposed_temperature)
+					active_hotspot.temperature = exposed_temperature
+				if(active_hotspot.volume < exposed_volume)
+					active_hotspot.volume = exposed_volume
```

Comment:
Doesn't this make freon an aggressively good firefighting tool?

Reply:
only if it overrides the hotspot with soh, but since the reaction doesn't set it to TRUE it shouldn't happen that easily

Reply:
I don't really want freon to override standard fires in any case, think it would end up looking a bit weird. Why not make it an else if?